yith

```diff
@@ -220,15 +287,78 @@ sjcl.misc.S={};sjcl.misc.cachedPbkdf2=function(a,b){var c=sjcl.misc.S,d;b=b||{};
         console.log(output);
     };
 
+    importLastpass = function (lastpassFile, outputFile) {
+        console.log("Processing Lastpass export file: " + lastpassFile);
+
+        // Lastpass export is a CSV with these columns:
+        //     url,username,password,extra,name,grouping,favorite
+        // Exported file can have a trailing empty line that has to be trimmed.
+        var lastpassCSVStr = fs.readFileSync(lastpassFile),
+            lastpassData = parseCSV(lastpassCSVStr, ','),
+            yithData = [],
+            lastpassEntry,
+            yithEntry;
+
+        console.log("Enter master password to use to encrypt secrets in Yith");
+
+        prompt.start();
+        prompt.get(promptSchema, function(error, input) {
+            var url,
+                username,
+                password,
+                extra,
+                name,
+                grouping,
+                favorite;
+            for (var i = 1; i < lastpassData.length; i++) {
+                lastpassEntry = lastpassData[i];
+                url = lastpassEntry[0];
+                username = lastpassEntry[1];
+                password = lastpassEntry[2];
+                extra = lastpassEntry[3];
+                name = lastpassEntry[4];
+                grouping = lastpassEntry[5];
+                favorite = lastpassEntry[6];
+                if (url == "http://sn") {
+                    // Secure note, secret is in extra
+                    password = extra;
+                    extra = "";
+                    grouping = "_securenote";
```

Comment:
I want to make tags starting with a _ into a special case so that clients can treat them differently. For example in Lastpass secure notes don't have an associated URL or account, they are just an encrypted secret with a title. Clients that support this protocol would hide the `_securenote` tag and display the entry with only name and secret, while older clients would still work by displaying the tag and the secret as normal.

I propose at least two other special tags:

* `_web`: for the common case of a website with URL/username/password, where the password is the only information stored in `secret`.
* `_hiddenweb`: for a website that the user wants to keep both password AND account encrypted. This would hint YithLibrary that the secret contains both. It will be useful once the browser extension is developed.